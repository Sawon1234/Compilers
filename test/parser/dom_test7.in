# test new function definition
fun add (a : Int, b : Int) : Int = a + b;
return add(5, 7);