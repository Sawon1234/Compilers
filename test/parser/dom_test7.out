fun add < > ( a : Int < > , b : Int < > ) : Int < > return a . plus < > ( b ) ; return add < > ( 5 , 7 ) ;